automation/run_ios_tests.sh

```diff
@@ -5,10 +5,10 @@ set -euvx
 ./megazords/ios-rust/build-xcframework.sh --build-profile release
 set -o pipefail && \
 xcodebuild \
-  -workspace ./megazords/ios-rust/MozillaTestServices/MozillaTestServices.xcodeproj/project.xcworkspace \
+  -project megazords/ios-rust/MozillaTestServices/MozillaTestServices.xcodeproj \
```

Comment:
The iOS test failure is a bit odd, but could it be due to this? Ideally it shouldn't since I don't think we reference another project but..XCode...